### ISSUE TITLE
`http` urls don't work with Bun plugin system

### ISSUE THREAD

**Issue by tr1ckydev** [oven-sh/bun#3679]
### What version of Bun is running?

0.6.14

### What platform is your computer?

Linux 6.4.3-1-cachyos x86_64 unknown

### What steps can reproduce the bug?

1. Create a plugin that can fetch the content of an http file url with text endpoint.
2. Import it.
```ts
import "./myPlugin.ts"
import data from "https://raw.githubusercontent.com/itsfoss/text-files/master/agatha.txt"

console.log(data);
```
3. Run the file.

### What is the expected behavior?

Console should log the contents inside the text file.

### What do you see instead?

Throws an error `error: FileNotFound reading "/itsfoss/text-files/master/agatha.txt"`.

**Comment by westtrade** [comment]
Also don't work command 
```
bun install https://cdn.sheetjs.com/xlsx-0.20.0/xlsx-0.20.0.tgz
```

**Comment by Jarred-Sumner** [comment]
This should be fixed in Bun v0.7.1 or Bun v0.7.0

please file a new issue with the namespace thing if it’s blocking you
